libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts

```diff
@@ -45,51 +47,113 @@ describe(WattNavListModule.name, () => {
+            redirectTo: 'default-page',
+          },
+          {
+            path: 'default-page',
+            component: TestPageComponent,
+          },
+          {
+            path: 'other-page',
+            component: TestPageComponent,
+          },
+        ],
+      }
+    );
 
-    const view = await render(TestAppComponent, {
-      declarations: [TestPageComponent],
-      imports: [WattNavListModule],
-      routes: [
-        {
-          path: '',
-          children: [
-            {
-              path: 'page-1',
-              component: TestPageComponent,
-            },
-            {
-              path: 'page-2',
-              component: TestPageComponent,
-            },
-          ],
-        },
-      ],
+    const activeClass = 'active';
+    const otherPageLink = await screen.findByRole('link', {
+      name: /other page/i,
     });
 
-    const expectedClass = 'active';
+    await view.navigate('/');
 
-    let link = view.getByRole('link', {
-      name: /page 2/i,
-    }) as HTMLAnchorElement;
+    expect(otherPageLink).not.toHaveClass(activeClass);
 
-    expect(Object.values(link.classList)).not.toContain(expectedClass);
+    await view.navigate(otherPageLink);
 
-    await view.navigate(link);
+    expect(otherPageLink).toHaveClass(activeClass);
+  });
 
-    link = view.getByRole('link', {
-      name: /page 2/i,
-    }) as HTMLAnchorElement;
+  it('Supports external links', async () => {
+    // Arrange
+    await render(
+      `
+      <watt-nav-list>
+        <watt-nav-list-item link="https://energinet.dk">
+          Energinet
+        </watt-nav-list-item>
+      </watt-nav-list>
+    `,
+      {
+        imports: [WattNavListModule],
+      }
+    );
+
+    // Act
 
-    expect(Object.values(link.classList)).toContain(expectedClass);
+    // Assert
+    expect(
+      await screen.findByRole('link', {
+        name: /energinet/i,
+      })
+    ).toHaveAttribute('href', 'https://energinet.dk');
+  });
+
+  describe(`${WattNavListModule.name} - Verify links added with href attribute opens in expected windows`, () => {
+    const setup = async (target: string | null) => {
+      // Arrange
+      await render(
+        `
+        <watt-nav-list>
+          <watt-nav-list-item link="https://energinet.dk" [target]="target">
+            Energinet
+          </watt-nav-list-item>
+        </watt-nav-list>
+      `,
+        {
+          imports: [WattNavListModule],
+          componentProperties: {
+            target,
+          },
+        }
+      );
+    };
+
+    it('Opens external links in a new browser tab, when specified', async () => {
+      // Arrange
+      await setup('_blank');
+      // Act
+      // Assert
+      const link = await screen.findByRole('link', {
+        name: /energinet/i,
+      });
+      expect(link).toHaveAttribute('target', '_blank');
+    });
+
+    it('Opens external links in the same browser tab by default', async () => {
+      await setup(null);
```

Comment:
nitpick: add *Arrange* comment to test setup
```suggestion
      // Arrange
      await setup(null);
```